keeparelease/keeparelease_test.go

```diff
@@ -0,0 +1,70 @@
+package keeparelease
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/lithammer/dedent"
+)
+
+const changelog string = `
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [[0.4.0]] - 2018-09-07
+
+### Added
+
+- A feature
+
+### Fixed
+
+- A fix
+
+## [[0.3.0]] - 2018-09-05
+
+### Added
+
+- Make the Git operations idempotent.
+- Implement all the operations required to create an environment.
+- Create an Octopus client to simplify the operation using its REST API.
+- Add --dry-run flag to bypass the write operations.
+
+[//]: # (Release links)
```

Comment:
Just because I'm curious, are the release links and issue/PR links part of the format? Are they per-release or just separate lists at the end of the file?